profile_library/mueller-licht/404094/model.json

```
@@ -0,0 +1,14 @@
+{
+  "calculation_strategy": "lut",
+  "created_at": "2025-01-25T17:25:15.201235",
+  "measure_description": "Measured with utils/measure script",
+  "measure_device": "TP-Link HS110",
+  "measure_method": "script",
+  "measure_settings": {
+    "SAMPLE_COUNT": 2,
+    "SLEEP_TIME": 3,
+    "VERSION": "v1.17.2:docker"
+  },
+  "name": "404094 - Tint Amela 30cm, white+ambiance (1800-6500K)",
```

We don't want to repeat the model id in the profile name.

```suggestion
  "name": "Tint Amela 30cm, white+ambiance (1800-6500K)",
```